lib/adapters/fetch.js

```diff
@@ -28,17 +28,20 @@ const encodeText = isFetchSupported && (typeof TextEncoder === 'function' ?
 
 const supportsRequestStream = isReadableStreamSupported && (() => {
   let duplexAccessed = false;
-
-  const hasContentType = new Request(platform.origin, {
-    body: new ReadableStream(),
-    method: 'POST',
-    get duplex() {
-      duplexAccessed = true;
-      return 'half';
-    },
-  }).headers.has('Content-Type');
-
-  return duplexAccessed && !hasContentType;
+  try {
+    const hasContentType = new Request(platform.origin, {
+      body: new ReadableStream(),
+      method: 'POST',
+      get duplex() {
+        duplexAccessed = true;
+        return 'half';
+      },
+    }).headers.has('Content-Type');
+  
+    return duplexAccessed && !hasContentType;
+  } catch () {
```

Comment:
This will throw an exception “_SyntaxError: Unexpected token ')'_”. Consider adding a binding identifier for the exception in `catch` 

```suggestion
  } catch (e) {
```